website/docs/advanced/config-v2-migration-guide.md

```diff
@@ -8,29 +8,33 @@ This guide will help you migrate from Config V1 to Config V2.
 
 ## What is Config V2?
 
-Config V2 is the next generation of ConfigCat. It comes with a new dashboard, API, SDKs, and features. It's not backward compatible with Config V1. Read more about the new features in the [Config V2 Overview](advanced/config-v2).
+Config V2 is the next generation of ConfigCat. It comes with a new dashboard, API, SDKs, and features. The V1 Dashboard, API, and SDKs are not compatible with V2. However the V2 SDKs are backwards compatible with V1.
+
+Read more about the new features in the [Config V2 Overview](advanced/config-v2).
 
 ## A few things to consider before migration
 
 - Config V2 and V1 are completely separate. They don't affect each other in any way. You can use them side by side.
-- Every newly created config is a V2 config.
+- After Config V2 launches, every newly created config will be a V2 config by default.
 - You can migrate your V1 configs to V2. That will mean that a V1 and a V2 version of your config will be available at the same time. The V2 version will have a new SDK key for each environment.
-- There is no automatic sync between the V1 and V2 configs. You have to manually update the V2 config if you make changes to the V1 config and vice versa.
-- Once started it's recommended to migrate your V1 configs to V2 as quickly as possible to avoid confusion.
+- There is no automatic sync between the V1 and V2 configs. If you want to keep them in sync, you have to manually update the V2 config when you make changes to the V1 config and vice versa.
+- Considering the above, it's recommended to migrate your V1 configs to V2 as quickly as possible to avoid confusion. However, there is no pressure to migrate. Although we have plans to phase out V1 eventually, you can stay on it for a long time.
 - There is no pressure to migrate. You can stay on V1 for a long time.
 
 ## Migrating from Config V1 to Config V2
 
+To migrate from Config V1 to Config V2, you have to create a new V2 config and update your applications to use the new config. The migration process is the following:
```

Comment:
A you have to create rész szerintem nem fedi így a valóságot, mert így én úgy tudom érteni, hogy csinálni kell egy új configot, de az egy másik flow.